web/src/reducers/activities.selectors.js

```diff
@@ -0,0 +1,76 @@
+import { createSelector } from 'reselect';
+import { selectApdData } from './apd.selectors';
+
+export const selectActivityKeys = ({ activities: { allKeys } }) => allKeys;
```

Comment:
First argument is state (destructured), and second is props (also destructured!).